global_costmap:

  map_type: costmap

  footprint: [[0.25, 0.21],
              [-0.25, 0.21],
              [-0.25, -0.21],
              [0.25, -0.21]]

  #transform_tolerance: 0.5
  inscribed_radius: 0.3
  circumscribed_radius: 0.32

  global_frame: /map
  robot_base_frame: /base_link
  update_frequency: 0.5
  publish_frequency: 0.1
  static_map: true
  rolling_window: false

  #Investigate what this actually does
  cost_scaling_factor: 10.0

  plugins:
  - {name: static_layer, type: 'costmap_2d::StaticLayer'}
#  - {name: obstacle_layer, type: 'costmap_2d::ObstacleLayer'}
  - {name: inflation_layer, type: 'costmap_2d::InflationLayer'}

  static_layer:
    map_topic: /map
    track_unknown_space: true
    unknown_cost_value: 255

#  obstacle layer:
#    observation_sources: base_scan
#    base_scan:
#      data_type: LaserScan
#    obstacle_range: 2.5
#    raytrace_range: 3.0

  inflation layer:
    inflation_radius: 0.1
